### PR TITLE
Added Gemstone TimeFilter to SPCTools

### PR DESCRIPTION
**Pull request by Tenacs**
package.json will need to be updated once common pages is published in gemstone

### CODE REVIEW

**Comment by clackner-gpa** [comment]
retargeted since this is related to the Gemstone update

File: Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/SelectStatisticsData.tsx [hunk 1]
```diff
@@ -40,38 +40,13 @@ declare let userIsAdmin: boolean;
...
-          return 30;
-        if (val === '90 Days')
-          return 90;
-        if (val === '180 Days')
-          return 180;
-        if (val === '365 Days')
-          return 365;
-        return 0;
-      }
+    type TimeUnit = 'y' | 'M' | 'w' | 'd' | 'h' | 'm' | 's' | 'ms'
```
Comment (gcsantos-gpa) [inline Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/SelectStatisticsData.tsx:49]: We should move this outside of the React component so it doesn't recreate every render.

File: Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx [hunk 2]
```diff
@@ -210,6 +210,7 @@ const WizardTest = () => {
         ])
     }
 
+    type TimeUnit = 'y' | 'M' | 'w' | 'd' | 'h' | 'm' | 's' | 'ms'
```
Comment (gcsantos-gpa) [inline Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx:213]: Same as above

**Review by gcsantos-gpa — COMMENTED** [review]
Minor nitpicks, I can approve and fix in https://github.com/GridProtectionAlliance/openXDA/pull/360

Edit: We also need conflicts fixed

**Review by gcsantos-gpa — APPROVED** [review]

**Pull request merged**